stock_picking_view_adj_asx/views/stock_picking_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+             <xpath expr="//field[@name='origin']" position="after">
+                <label for="evidence_status" string="Evidence Status"/>
+                <div>
+                    <field name="evidence_status"/>
+                    <div>
+                        <field name="evidence_text"/>
+                    </div>
+                </div>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="vpicktree" model="ir.ui.view">
```

Comment:
In case we want to show these fields for "Receipts" only, we can do something like the following:
https://github.com/qrtl/hls-custom/blob/08f6151ee318248208fb82cc3fd0186791747b9d/stock_picking_delivery_due_date/models/stock_picking.py#L32-L43
https://github.com/qrtl/hls-custom/blob/08f6151ee318248208fb82cc3fd0186791747b9d/stock_picking_delivery_due_date/views/stock_picking_views.xml#L16-L26

We may not do this this time to keep the changes minimal.